package.json

```diff
@@ -824,11 +824,13 @@
     "azure-iot-device-mqtt": "^1.15.0",
     "azure-iothub": "^1.12.4",
     "body-parser": "^1.18.2",
+    "dummy-json": "^3.0.4",
```

Comment:
Package was a dependency in `external_lib` but this way we can manage its version and dependencies in `package.json`